tests/manage/z_cluster/cluster_expansion/test_add_non_ocs_workers.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import tier1, ManageTest
+from ocs_ci.ocs.node import add_nodes, taint_nodes, get_worker_nodes_not_in_ocs
+from ocs_ci.ocs.resources.pod import get_pod_node, get_plugin_pods
+
+logger = logging.getLogger(__name__)
+
+
+@tier1
+class TestAddNonOcsNodes(ManageTest):
+    """
+    Checks for plugin pods toleration come-up on newly added worker nodes
+    """
+
+    def test_add_capacity_node_restart(self, pvc_factory, pod_factory):
```

Comment:
We are not doing add-capacity not node restarts, please correct the test function name.

Reply:
done